index.rst

```diff
@@ -3,11 +3,6 @@
 Symfony Documentation
 =====================
 
-.. toctree::
-    :hidden:
-
-    changelog
```

Comment:
The changelog would not be generated (you can see it [here](http://symfony.com/doc/current/changelog.html)) if this were removed, right?